CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -70,12 +70,15 @@ Some background and text colors use the `GiniColor` type with which you can set
 	- Title &#8594; <span style="color:#009EDF">*ginicapture.navigationbar.camera.help*</span> localized string
 
 ##### 2. Camera preview
+- Preview frame color &#8594;  `GiniConfiguration.cameraPreviewFrameColor`
 - Guides color &#8594;  `GiniConfiguration.cameraPreviewCornerGuidesColor`
 - Focus large image &#8594; <span style="color:#009EDF">*cameraFocusLarge*</span> image asset
 - Focus large small &#8594; <span style="color:#009EDF">*cameraFocusSmall*</span> image asset
 - Opaque view style (when tool tip is shown)  &#8594;  `GiniConfiguration.toolTipOpaqueBackgroundStyle`
 
 ##### 3. Camera buttons container
+- Background color &#8594;  `GiniConfiguration.cameraButtonsViewBackgroundColor`
+- Container view background color under the scroll indicator  &#8594;  `GiniConfiguration.cameraContainerViewBackgroundColor` 
```

Comment:
Where is there a scroll indicator on the camera screen? 🤔

Reply:
![IMG_6653](https://user-images.githubusercontent.com/3911816/166957312-a8df4823-ac77-46a4-87a6-9577aa5277a1.jpg)
Bottom area with `scroll indicator`.

Reply:
That one is called the `home indicator`: https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/

Reply:
Thanks. I'll fix it.